src/Android/Xamarin.Android/Samples/Layers/RasterLayerFile/RasterLayerFile.cs

```diff
@@ -77,7 +77,7 @@ private void CreateLayout()
             Viewpoint myFullRasterExtent = new Viewpoint(myRasterLayer.FullExtent);
 
             // Zoom to the extent
-            _myMapView.Map.InitialViewpoint = myFullRasterExtent;
+            _myMapView.SetViewpoint(myFullRasterExtent);
```

Comment:
Turns out this doesn't work on Android either. I'm not sure if the [suggested workarounds for XF](https://github.com/Esri/arcgis-runtime-samples-dotnet/pull/364#discussion_r154547385) will work here — you'll have to give it a try.